src/trio/_core/_tests/test_run.py

```diff
@@ -456,7 +457,8 @@ async def crasher() -> NoReturn:
                 # nursery block exited, all cancellations inside the
                 # nursery block continue propagating to reach the
                 # outer scope.
-                assert len(multi_exc.exceptions) == 4
+                # the noqa is for "Found assertion on exception `multi_exc` in `except` block"
+                assert len(multi_exc.exceptions) == 4  # noqa: PT017
```

Comment:
oh derp, I should've thought of this one myself. You can leave this one as a `noqa` though and let me handle it in the multierror PRs